src/content/1.7/development/architecture/domain/cqrs.md

```diff
@@ -30,6 +30,20 @@ Fortunately, by implementing CQRS it allows PrestaShop to quickly build new API,
 
 {{< figure src="../img/new_architecture.png" title="New architecture using CQRS" >}}
 
+## Validation
```

Comment:
Im not sure its the best way... 
Maybe this needs a separate page, which would describe the flow of all the validation instead? (just idk where to put it).